crmsh/utils.py

```diff
@@ -397,7 +397,53 @@ def chown(path, user, group):
     else:
         import grp
         gid = grp.getgrnam(group).gr_gid
-    os.chown(path, uid, gid)
+    try:
+        os.chown(path, uid, gid)
+    except os.error as err:
+        cmd = "sudo chown {}:{} {}".format(user, group, path)
+        rc, out, err = get_stdout_stderr(cmd, no_reg=True)
+        if rc != 0:
+            fatal("Failed to chown {}: {}".format(path, err))
+
+
+def chmod(path, mod):
+    try:
+        os.chmod(path, mod)
+    except os.error as err:
+        cmd = "sudo chmod {} {}".format(format(mod,'o'), path)
+        rc, out, err = get_stdout_stderr(cmd, no_reg=True)
+        if rc != 0:
+            fatal("Failed to chmod {}: {}".format(path, err))
+
+
+def touch(file_name):
+    rc, out, err = get_stdout_stderr("touch " + file_name, no_reg=True)
+    if rc != 0:
+        rc, out, err = get_stdout_stderr("sudo touch " + file_name, no_reg=True)
+        if rc != 0:
+            fatal("Failed create file {}: {}".format(file_name, err))
+
+
+def copy_local_file(src, dest):
+    try:
+        shutil.copyfile(src, dest)
+    except os.error as err:
+        rc, out, err = get_stdout_stderr("sudo cp {} {}".format(src, dest), no_reg=True)
+        if rc != 0:
+            fatal("Failed to copy file from {} to {}: {}".format(src, dest, err))
+
+
+def rmfile(path, ignore_errors=False):
+    """
+    Try to remove the given file, and
+    report an error on failure
+    """
+    try:
+        os.remove(path)
+    except os.error as err:
+        rc, out, err = get_stdout_stderr("sudo rm -rf " + path, no_reg=True)
```

Comment:
Argument `-rf` should not be specified since `os.remove` is neither recursive nor forced.